docs/_snippets/installation/plugins-mapping/plugins-mapping.html

```diff
@@ -0,0 +1,848 @@
+            </td>
+            <td>
+               <p style="margin-left:0px;"><a href="https://ckeditor.com/docs/ckeditor5/latest/features/toolbar/toolbar.html">Editor toolbar</a></p>
+            </td>
+         </tr>
+         <tr>
+            <td>
+               <p style="margin-left:0px;"><a href="https://ckeditor.com/cke4/addon/uicolor">uicolor</a></p>
+            </td>
+            <td>
+               <p style="margin-left:0px;"><a href="https://ckeditor.com/docs/ckeditor5/latest/framework/guides/deep-dive/ui/theme-customization.html">Theme customization</a></p>
+            </td>
+         </tr>
+         <tr>
+            <td>
+               <p style="margin-left:0px;"><a href="https://ckeditor.com/cke4/addon/undo">undo</a></p>
+            </td>
+            <td>
+               <p style="margin-left:0px;"><a href="https://ckeditor.com/docs/ckeditor5/latest/api/module_undo_undo-Undo.html">Undo</a></p>
+            </td>
+         </tr>
+         <tr>
+            <td>
+               <p style="margin-left:0px;"><a href="https://ckeditor.com/cke4/addon/uploadfile">uploadfile</a></p>
+            </td>
+            <td>N/A</td>
+         </tr>
+         <tr>
+            <td>
+               <p style="margin-left:0px;"><a href="https://ckeditor.com/cke4/addon/uploadimage">uploadimage</a></p>
+            </td>
+            <td>
+               <p style="margin-left:0px;"><a href="https://ckeditor.com/docs/ckeditor5/latest/features/images/image-upload/image-upload.html">Image upload</a></p>
+            </td>
+         </tr>
+         <tr>
+            <td>
+               <p style="margin-left:0px;"><a href="https://ckeditor.com/cke4/addon/uploadwidget">uploadwidget</a></p>
+            </td>
+            <td>
+               <p style="margin-left:0px;"><a href="https://ckeditor.com/docs/ckeditor5/latest/features/images/image-upload/image-upload.html">Image upload</a></p>
+            </td>
+         </tr>
+         <tr>
+            <td>
+               <p style="margin-left:0px;"><a href="https://ckeditor.com/cke4/addon/widget">widget</a></p>
+            </td>
+            <td>
+               <p style="margin-left:0px;"><a href="https://ckeditor.com/docs/ckeditor5/latest/api/module_widget_widget-Widget.html">Widget</a></p>
+            </td>
+         </tr>
+         <tr>
+            <td>
+               <p style="margin-left:0px;"><a href="https://ckeditor.com/cke4/addon/widgetselection">widgetselection</a></p>
+            </td>
+            <td>
+               <p style="margin-left:0px;"><a href="https://ckeditor.com/docs/ckeditor5/latest/api/module_widget_widget-Widget.html">Widget core</a></p>
+            </td>
+         </tr>
+         <tr>
+            <td>
+               <p style="margin-left:0px;"><a href="https://ckeditor.com/cke4/addon/wysiwygarea">wysiwygarea</a></p>
+            </td>
+            <td>Core</td>
```

Comment:
Hm, this one is tricky. 

On one hand, there's no iframe-based editor implementation in v5. v5 is more like [Div Editing Area](https://ckeditor.com/cke4/addon/divarea) and editor implementations use `InlineEditableUIView` from ckeditor5-ui to create the editing root.

OTOH, this was the most popular way to create and editor in 4, so instead N/A it's worth mentioning that this is a core feature (although no longer iframe-based).